### PR TITLE
Provide default value for version

### PR DESCRIPTION
**Pull request by turion**
It would be cool if one doesn't have to specify the version as an input.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,3 +49,15 @@ jobs:
...
         run: cabal new-test all
+  test-generate-matrix-default:
+    name: "Tests whether generating the matrix from cabal works with a default version"
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Extract the tested GHC versions
+        id: set-matrix
+        uses: turion/get-tested@v0.1.6.0
```
Comment (turion) [inline .github/workflows/ci.yml:60]: This needs to be changed back at the end.

**Comment by Kleidukos** [comment]
hmm I don't agree too much with this, it makes things easier to have the version of the tool written down somewhere. Moreover, right now the version of the tool and the action are the same by accident, I will think of moving the action elsewhere (in @haskell-actions for instance), and then it will be a totally different number.

**Comment by turion** [comment]
You are definitely right there in principle. My motivation was the following: I like and use `get-tested` a lot, but one common source of errors was that I used a newer GHC version than my pinned `get-tested` version supported. If the version of the _action_ is bumped every time the version of the _tool_ is bumped, then dependabot can automatically bump that version in my local usage of the action. If you want to decouple the two, you would have double the maintenance effort by bumping two versions.

**Comment by Kleidukos** [comment]
Ah yes that was a problem of fixed known GHC versions, but I've removed this behaviour: https://github.com/Kleidukos/get-tested/commit/3e5545ecf9db74bb16f15e408e1e60dae3566bfa

**Comment by Kleidukos** [comment]
(or perhaps were you thinking of something else?)

**Comment by turion** [comment]
I just tested and it seems that is exactly the thing :D ok, in that case my workflow doesn't depend so crucially on the `get-tested` version anymore and all is well. Thanks!

**Pull request closed without merging**